Source/AtomicJS/Javascript/JSAPI.cpp

```diff
@@ -69,13 +69,17 @@ void js_class_declare_internal(JSVM* vm, void* uniqueClassID, const char* packag
         duk_put_prop(ctx, -3);
         duk_pop(ctx);
     }
-    else
+    
+    if (!duk_get_global_string(ctx, package))
     {
-        // RefCounted only supported in Atomic package
-        assert(String("Atomic") == package );
+        // Failure leaves undefined on the stack
```

Comment:
Ok, I filed an issue to define when the the package object is created.  It has been in "user code", though I will be moving it the package init as generated by JSBind, there are some details in the issue: https://github.com/AtomicGameEngine/AtomicGameEngine/issues/133